tests/unit/lms/services/lti_h_test.py

```diff
@@ -23,8 +21,7 @@ def test_it_does_nothing_if_the_feature_isnt_enabled(
 
         lti_h_svc.upsert_course_group()
 
-        hapi_svc.patch.assert_not_called()
-        hapi_svc.put.assert_not_called()
+        h_api.update_group.assert_not_called()
```

Comment:
These tests make much more sense now, rather than making generic assertions about patch and put